assets/src/adaptivity/scripting.ts

```diff
@@ -57,25 +57,15 @@ export const getExpressionStringForValue = (
 
     // it might be CSS string, which can be decieving
     let actuallyAString = false;
+    const expressions = extractAllExpressionsFromText(val);
+    // A expression will not have a ';' inside it. So if there is a ';' inside it, it is CSS.
+    const isCSSString = expressions.filter((e) => e.includes(';'));
+    if (isCSSString?.length) {
+      actuallyAString = true;
+    }
     try {
       const testEnv = new Environment(env);
-      const evalResult = evalScript(`let foo = ${val};`, testEnv);
```

Comment:
Removing all the previous CSS related fixes. We don't need them anymore. I have test few CSS related tickets and they we working properly.